src/pages/settings/InitialSettingsPage.js

```diff
@@ -128,6 +128,8 @@ const defaultProps = {
     ...withCurrentUserPersonalDetailsDefaultProps,
 };
 
+const INBOX_URL = 'inbox';
```

Comment:
Can we move this to CONST and also replace it in places like this one? https://github.com/Expensify/App/blob/c0313c2246608ac095488d0916cd2a22ea5bd610/src/pages/settings/Wallet/ExpensifyCardPage.js#L211

Reply:
Thanks, done :)